framework/decode/vulkan_replay_dump_resources_compute_ray_tracing.cpp

```diff
@@ -1228,18 +1414,16 @@ VkResult DispatchTraceRaysDumpingContext::DumpImmutableDescriptors(uint64_t qs_i
             }
         }
     }
-    else
+    else if (!is_dispatch)
```

Comment:
Isn't `!is_dispatch` implied by the `else` here?

Reply:
yes